plugins/git/git.plugin.zsh

```diff
@@ -221,6 +221,7 @@ alias glp="_git_log_prettily"
 
 alias gm='git merge'
 alias gmom='git merge origin/$(git_main_branch)'
+alias gmod='git merge origin/develop'
```

Comment:
This would need to use the function `git_develop_branch` like the other aliases
```suggestion
alias gmod='git merge origin/$(git_develop_branch)'
```